src/objects/exchangeable_obj.f90

```diff
@@ -36,10 +37,10 @@ module subroutine const(this, grid, metadata, forcing_var)
     if (err /= 0) stop "exchangeable:dqdt_3d: Allocation request failed"
     this%data_3d = 0
 
-    allocate( this%halo_south_in( grid%ns_halo_nx+halo_size*2, grid%halo_nz,   halo_size    )[*])
-    allocate( this%halo_north_in( grid%ns_halo_nx+halo_size*2, grid%halo_nz,   halo_size    )[*])
-    allocate( this%halo_east_in(    halo_size,     grid%halo_nz, grid%ew_halo_ny+halo_size*2)[*])
-    allocate( this%halo_west_in(    halo_size,     grid%halo_nz, grid%ew_halo_ny+halo_size*2)[*])
+    allocate( this%halo_south_in( grid%ns_halo_nx+halo_size*2, grid%halo_nz,   halo_size+1      )[*])
```

Comment:
@d-reynolds

Reply:
This is because for exchanging U/V, we need to sometimes need to exchange rows/columns and not just one. So for all over exchangeables, and for the default exchangeable_object routines, the standard halo_size is still correct.